### PR TITLE
Alex Schwartz -- Independent Challenge 1

### PR DESCRIPTION
**Pull request by aschwartz1**
https://github.com/turingschool-examples/schoolhouse_rock

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    split_start_time = @start_time.split(':')
+
+    # Get start hour
+    start_hour = split_start_time[0].to_i
+
+    # Calculate end hour
+    end_hour = start_hour + @hours_in_school_day
+
+    # Create end time string
+    end_time = "#{end_hour}:#{split_start_time[1]}"
```
Comment (memcmahon) [inline lib/school.rb:27]: You don't need to assign this to a variable, you could simply end the method with:

`"#{end_hour}:#{split_start_time[1]}"`

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    # Separate the time on a colon
+    split_end_time = end_time.split(':')
+
+    # Get end hour
+    end_hour_24 = split_end_time[0].to_i
+
+    # Convert end hour to 12hr format (it's normally 24hr format)
+    end_hour_12 = (end_hour_24 <= 12) ? end_hour_24 : end_hour_24 % 12
+
+    converted_end_time = "#{end_hour_12}:#{split_end_time[1]}"
```
Comment (memcmahon) [inline lib/school.rb:40]: Same note as above :)

File: test/school_test.rb [hunk 3]
```diff
@@ -45,11 +40,36 @@ def test_it_can_add_student_names
...
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  def test_is_full_time?
+    school1 = School.new('9:00', 7)
+    school2 = School.new('9:00', 3)
+
+    assert_equal true, school1.is_full_time?
+    assert_equal false, school2.is_full_time?
```
Comment (memcmahon) [inline test/school_test.rb:55]: Great job confirming both possibilities

**Review by memcmahon — COMMENTED** [review]
🎉 Great Job!!

Score: 4

File: lib/school.rb (hunk 1 above)
Comment (aschwartz1) [reply lib/school.rb:27]: Roger that. It feels so weird to have an operation just hanging out on that line, all by its lonesome!  But on the other hand we don't need a variable for something we won't ever reference...

Thanks!

**Pull request closed without merging**